tools/lint-all

```diff
@@ -83,9 +83,9 @@ def check_pep8(files):
     failed = False
     ignored_rules = [
         'E402', 'E501', 'W503', 'E711', 'E201', 'E203', 'E202', 'E128', 'E226', 'E124', 'E125',
-        'E126', 'E127', 'E121', 'E122', 'E123', 'E266', 'E265', 'E261', 'E701', 'E221', 'E303',
+        'E126', 'E231', 'E121', 'E122', 'E123', 'E266', 'E265', 'E261', 'E701', 'E221', 'E303',
```

Comment:
Yes, As the task say you don't want to reflow the lines, just removing the E127 code is sufficient. The line will be a little shorter but that is fine.